### DISCUSSION TITLE
How to get achievement galaxy brain? ( tutorial for YouTube ) #1

### DISCUSSION THREAD

**Discussion by fugoou** [community/community#72932] [GitHub Education]
### Hi everyone,

e.g how #1

**Comment by Git-SalahEdd** [comment]
You have to answer 2 discussion posts where the original poster marks your reply as an answer. It may take a lot more than 2 total discussion posts answered if your replies are not marked as the answer. It depends on whether or not your replies are marked as the answer. 2 replies marked as an answer is the base amount you need. Bronze is 8 replies marked as an answer. Silver is 16 replies marked an an answer. Gold is 32 replies marked as an answer.
